scripts/publish-cdn.sh

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env bash
+
+set -e   # exit on error
+
+# TODO: Update script!
+
+VERSION=$(node scripts/version.js)
+VERSION_TAG=v${VERSION:0:1}.x
+echo $@
+if aws s3 ls "redocly-cdn/redoc/v$VERSION/" "$@"; then
+  echo "Version $VERSION already exists"
+  exit 1
+else
+  echo Releasing $VERSION
+
+  echo Uploading to S3 $VERSION
+  aws s3 cp bundles "s3://redocly-cdn/redoc/v$VERSION/bundles" --recursive "$@"
+  aws s3 cp typings "s3://redocly-cdn/redoc/v$VERSION/typings" --recursive "$@"
+  aws s3 cp CHANGELOG.md "s3://redocly-cdn/redoc/v$VERSION/CHANGELOG.md" "$@"
+  aws s3 cp LICENSE "s3://redocly-cdn/redoc/v$VERSION/LICENSE" "$@"
+  aws s3 cp package.json "s3://redocly-cdn/redoc/v$VERSION/package.json" "$@"
+  aws s3 cp README.md "s3://redocly-cdn/redoc/v$VERSION/README.md" "$@"
```

Comment:
We might want to extract these similar pieces of script into [bash functions](https://ryanstutorials.net/bash-scripting-tutorial/bash-functions.php).

Reply:
Good idea. done